fga-eps-mds/2020.1-stay-safe-front-end
# 52 code climate issues

## Issue 
Resolves #52 


## Description
Refactoring to remove some duplicated code and some others code-climate issues.

## Checklist:



## How to validate?
This PR changes a lot of differente files, the better way to validate is to see if the score of code-climate has increased and run the app and test some features to see if doesn't break anything (since it doesn't have tests implemented).
